src/components/UserCard/UserCard.tsx

```diff
@@ -9,7 +8,14 @@ import Avatar from 'components/Avatar';
 
 import './UserCard.css';
 
-const UserCard = (props) => {
+type UserCardProps = {
+  name: string;
+  user: any;
```

Comment:
Can you add a `// ToDo` here so we don't forget it? I'm ok with leaving it as `any` for now, but in the future we could use a proper type.